config/lib/express.js

```diff
@@ -141,7 +141,7 @@ module.exports.initModulesConfiguration = function (app, db) {
  */
 module.exports.initHelmetHeaders = function (app) {
   // Use helmet to secure Express headers
-  var SIX_MONTHS = 15778476000;
+  var SIX_MONTHS = 15778476;
```

Comment:
Also it'll be useful for anyone modifying this to have a short comment mentioning it's seconds.